finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java

```diff
@@ -30,7 +30,7 @@
   private InventoryList invList = new InventoryList();
   private SystemClient systemClient = new SystemClient();
 
-  @Timed(name = "inventoryPropertiesRequestTime", absolute = true,
```

Comment:
I wonder why the absolute is removed ? -> this is important for the output.

Reply:
Then can you explain why we use `absolute` here in the readme?